src/OpenTelemetry.Instrumentation.GrpcCore/OpenTelemetry.Instrumentation.GrpcCore.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="[3.15.0,4.0)" />
-    <PackageReference Include="Grpc.Core.Api" Version="[2.23.0,3.0)" />
+    <PackageReference Include="Grpc.Core.Api" Version="[2.43.0,3.0)" />
```

Comment:
I would consider even 2.46.6.
It is the latest (security fixed) version of `Grpc.Core` package. Nobody should use any older version. Can be done separately. Ref: https://groups.google.com/g/grpc-io/c/OTj5mb1qzb0 It can stay as you proposed.

Reply:
Makes sense, will create another PR for this